src/PanoViewer/PanoViewer.js

```diff
@@ -325,11 +325,10 @@ export default class PanoViewer extends Component {
 		 */
 
 		/**
-		 * Events that is fired when PanoViewer is ready to go.
-		 * @ko PanoViewer 가 준비된 상태에 발생하는 이벤트
+		 * Events that is fired when PanoViewer is ready to handle user interaction
+		 * @ko PanoViewer 가 유저와 상호작용을 시작할때 발생
```

Comment:
@happyhj I think it's exact status of PanoViewer. But As a user of PanoVIewer, he is not interected with interaction time. It would be better to say like ... "webgl is intiailized and it's ready to render".